+++
widget = "blank"  # See https://sourcethemes.com/academic/docs/page-builder/
headless = true  # This file represents a page section.
active = true # Activate this widget? true/false
weight = 1  # Order that this section will appear.
title = ""
subtitle = ""

[design]
  columns = "1" # Choose how many columns the section has. Valid values: 1 or 2.

[design.background]
  # Apply background color, gradient, or image. Any HTML color name or Hex value is valid.
  # color = "navy" # Background color.
  # gradient_start = "DeepSkyBlue" # Background gradient.
  # gradient_end = "SkyBlue" # Background gradient.
  image = ""  # Name of image in `static/img/`. # Background image.
  image_darken = 0.6  # Darken image? Range 0-1 where 0 is transparent and 1 is opaque.
  text_color_light = false # Text color (true=light or false=dark).

[design.spacing]
  padding = ["20px", "200px", "20px", "200px"] # Customize the section spacing. Order is top, right, bottom, left.

[advanced]
 css_style = "" # Custom CSS. 
 css_class = "mini" # CSS class.
+++

## The Origin of Species 
It is an exciting time to study evolutionary biology and speciation. As a research field that is largely focused on historical events, modern high-throughput sequencing approaches offer unprecedented opportunities to make inferences about said events. This is particularly true for non-model systems with limited experimental options. In combination with theoretical and particularly computational advances, we can now obtain genomes and then attempt to read them ["like a history book"](https://science.sciencemag.org/content/358/6368/1265.2) for any organism. __My research focuses on using genomics to advance our understanding of speciation__, and I have been working on this in a variety of non-model organisms: crows, cichlid fish, and lemurs.

{{< figure src="studyspecies.jpg" width="850px" caption="Hooded and Carrion Crows (left), Lake Ejagham cichlids (center), and gray and reddish-gray mouse lemurs (right). Isn't it weird that only the bird names are capitalized?" >}}

The origin of species has always interested me. As a traveling naturalist, it is easy to start wondering about processes behind patterns in geographic distributions and ecology, dissimilarities _and_ similarities between closely related species, and what keeps them apart. Or what evidently doesn't, when you encounter hybrids or clinal variation. I was slow to gain an interest in genetics as an undergrad, and the penny only really dropped during an MSc research project on song divergence as a behavioral mechanism of isolation between two hybridizing subspecies of the Gray-breasted Wood-Wren (see [Dingle et al. 2010](/publication/dingle-asymmetric-2010/index.html) and [Halfwerk et al. 2016](/publication/halfwerk-sharp-2016/index.html)). These birds were morphologically highly similar and vocally divergent, but since their songs are learned, we needed genetic tests to learn the subspecific identity of individual birds. At the same time, this genetic work also revealed high levels of divergence between these subspecies, despite looking nearly identical.

## Diversification in Tiny Primates
Mouse lemurs (genus _Microcebus_) are the smallest primates on earth and are endemic to Madagascar --- as is the entire so-called superfamily of lemurs. Until recently, they were thought to comprise only two species, but genetic work has shown that there is much diversity hidden within these highly cryptic, nocturnal creatures. At Duke University, in the [lab](http://yoderlab.org/) of my postdoc adviser [Anne Yoder](https://biology.duke.edu/people/anne-daphne-yoder) and -- among others -- collaborators [George Tiley](http://yoderlab.org/people/current-lab-members/george-tiley/) and [Jordi Salmona](https://scholar.google.com/citations?user=5-mQGREAAAAJ), I have mostly worked on a large and growing RADseq dataset of all 24 presently described species of mouse lemur to investigate phylogeny, patterns of diversification, demographic histories, and gene flow within and among species.

We have a [preprint](/publication/schusler-cryptic-2019/index.html) of a study of five species of mouse lemurs up, which confirms a lineage hitherto called "M. sp. nova #3" as a highly distinct taxon while calling into question the distinctness of two other species. We estimated rates of gene flow (finding, for instance, evidence for mitochondrial introgression) and divergence times. Interestingly, two pairs of these species occur sympatrically without ongoing gene flow, despite having diverged well under a million years ago.

{{< figure src="lemur_demography.png" width="800px" caption="Demographic histories for several mouse lemurs species. __A-C:__ Divergence times (y-axis) and effective population sizes (x-axis) inferred by BPP (A; no gene flow) and G-PhoCS without (B) and with (C) gene flow. __D-E__: Comparison of point estimates and 95% HPD for divergence times and effective population sizes for each node and lineage, respectively. __F:__ Effective population sizes (N<sub>e</sub>) through time for as inferred by MSMC for whole-genome data from a single individual (blue line), and by G-PhoCS for RADseq data without ('RAD: iso', yellow lines) and with ('RAD: mig', blue lines) gene flow. Abbreviations: 'msp3' = _M. sp. #3_, 'mac' = _M. macarthurii_, 'sim' = _M. simmonsi_, 'mit' = _M. mittermeieri_ and 'leh' = _M. lehilahytsara_.">}}

Along similar lines, I have found that previously suspected hybridization between two other mouse lemur species, which also diverged less than 1 ma ago, is not supported by RADseq data --- and the lack of hybridization despite co-occurrence implies that these species appear to have rapidly evolved reproductive isolation (unpublished data, with [Jörg Ganzhorn](https://www.biologie.uni-hamburg.de/en/forschung/oekologie-biologische-ressourcen/tieroeknatsch/mitarbeiter/jganzhorn.html)). Other ongoing mouse lemur projects include an investigation of genomic divergence associated with local adaptation in reddish-gray mouse lemur (_M. griseorufus_), modeling divergence and gene flow among all close sister species, and creating a dated phylogeny for all described species. Finally, we have a [preprint](/publication/campbell-pedigree-based-2019/index.html) up on an estimation of the _de novo_ mutation rate for the gray mouse lemur (_M. murinus_) based on deep whole-genome sequencing of a known pedigree of mouse lemurs from the [Duke Lemur Center](https://lemur.duke.edu/).

## Sympatric Speciation with a Twist
Speciation is often assisted by geographic isolation of populations, after dispersal or vicariance events. Mutation and drift will inevitably lead to genetic divergence when populations are physically separated, while selection will have to overcome gene flow in the absence of geographic isolation. This was generally considered extremely hard if not impossible throughout much of the 20th century. The discovery of several radiations of cichlid fish in isolated Cameroon crater lakes _with, within each lake, monophyletic mtDNA_ ([Schliewen, Tautz & Pääbo 1994](https://www.nature.com/articles/368629a0)), marked a real turning point. A flurry of subsequent theoretical work and a search for additional empirical examples ensued, with both lines of inquiry generating additional support that speciation can happen in sympatry. At this point, sympatric speciation is hardly a controversial topic anymore (even though it is generally thought to be uncommon at best). However, we have only recently gained the ability to assess monophyly of sympatric radiations -- a telltale sign of sympatric speciation -- not just for single genes, but for the entire genome. This is important because signals from individual genes can be highly misleading due to lineage sorting and gene flow.

With all of this in mind, I studied whole-genome sequences of the abovementioned Cameroon crater lake cichlids for a year in [Chris Martin's Lab](http://ib.berkeley.edu/labs/martin/) (then at UNC Chapel Hill, North Carolina, now at UC Berkeley, California), working together with PhD student [Emilie Richards](https://emiliejrichards.wixsite.com/emiliejrichards). This research supported that divergence took place _within_ the tiny crater lakes, i.e. in sympatry, but also suggested that the lakes do not appear to be as isolated as previously thought, given that some secondary gene flow from riverine populations occurred. For one radiation ([Poelstra et al. 2018](/publication/poelstra-speciation-2018/index.html)), we additionally found some support for a role of said gene flow in actually _promoting speciation_, while we did not in another ([Richards et al. 2018](/publication/richards-dont-2018/index.html)).

{{< figure src="cichlids.svg" width="1050px" caption="Monophyly (panels A-C) _and_ gene flow (panels D-E) in Lake Ejagham cichlids. The green and blue species are endemic to Lake Ejagham, and the orange and red species occur in nearby rivers. __A:__ An ML phylogeny. __B:__ A Phylogenetic network using Splitstree Neighbornet. __C:__ The distributions of, and two examples of, local genomic phylogenies, using [Saguaro](http://saguarogw.sourceforge.net/) cacti. __D:__ D-statistics. __E:__ An example of an introgressed segment detected using the f<sub>d</sub> statistic. Modified after [Poelstra et al. 2018](/publication/poelstra-speciation-2018/index.html)." >}}

## The Curious Case of Crows
Hybrid zones have long been a central locus of speciation research, and are commonly characterized as "natural
laboratories of evolution" or "windows into the evolutionary process." One reason for this is that hybrid zones capture an
intermediate stage of speciation, in which reproductive isolation is present but incomplete. A hybrid zone between partially grey (Hooded) and all-black (Carrion) crows in Europe had already been mapped by 1928 ([Meise](https://scholar.google.com/scholar_lookup?title=Die%20verbreitung%20der%20aaskr%C3%A4he%20%28Formenkreis%20Corvus%20corone%20L%29&journal=J%20F%C3%BCr%20Ornithologie&volume=76&pages=1-206&publication_year=1928&author=Meise%2CW)) and was discussed by as a key example of speciation and hybridization by Ernst Mayr mid-century --- and subsequently in several modern textbooks on evolution. However, little research had been conducted on this system and had perhaps raised more questions than answers, such as the intriguing finding of an apparent lack of (neutral) genetic differentiation between them.

For my PhD, advised by [Jochen Wolf](http://www.evol.bio.lmu.de/people/group_leaders/wolf/index.html) (then at Uppsala, Sweden, now in Munich, Germany), and with much of the work in collaboration with [Nagarjun Vijay](https://bio.iiserb.ac.in/faculty_profile.php?id=MTY=&lname=bmFnYXJqdW4=) and [Christen Bossu](https://www.ioes.ucla.edu/person/christen-bossu/), I looked into this system from a molecular perspective. Initially using candidate coloration genes ([Poelstra et al. 2013](/publication/poelstra-extensive-2013/index.html)), and then RNA-seq ([Poelstra et al. 2015](/publication/poelstra-transcriptomics-2015/index.html), and see also [Vijay et al. 2013](/publication/vijay-challenges-2013/index.html)) and whole-genome sequencing ([Poelstra et al. 2014](/publication/poelstra-genomic-2014/index.html)). One might argue that once again, more questions were raised than answered, but we generated some interesting findings nevertheless...

{{< figure src="climbing.png" width="750px" caption="Field work! To monitor crow nests, we eventually switched from climbing to using a drone. Note Schrödinger's climber in (1) and (2). Modified after [Weissensteiner et al. 2015](/publication/weissensteiner-low-budget-2015/)." >}}

For one, Carrion and Hooded Crows are nearly identical across almost the entire genome. This echoes those earlier findings, but is incredibly striking nevertheless, given that phenotypically, they nearly behave as "good species." Additionally, we found that _within_ Carrion Crows, as well within the phenotypically similar _orientalis_ crows, much higher levels of differentiation can be found. We could also quantify differentiation at a high resolution, finding _only 83 fixed differences_ between Carrion and Hooded Crows them across our entire genome assembly, all but one of which are in a single cluster in the genome. Differentiation in gene _expression_ mirrored this lack of differentiation, with substantial differences _only_ in the bases of feathers from the same body area that differ in color between Carrion and Hooded Crows. Finally, we got a long way towards identifying the genetic changes responsible for these differences in plumage coloration (see also [Vijay et al. 2016](/publication/vijay-evolution-2016/index.html) and [Knief et al. 2019](/publication/knief-epistatic-2019/index.html) for increased resolution on that).

{{< figure src="crows.png" width="750px" caption="__A:__ A circos plot showing the large degree of synteny between the Hooded Crow and Zebra Finch. __B:__ A PCA shows some unusual patterns with little variation in Hooded (_cornix_) Crows, little differentiation between Hooded <-> Carrion (_corone_) as well as Hooded <-> _orientalis_ Crows, and substantial differentiation _within_ Carrion and _orientalis_ crows. __C:__ Only a single region in the genome is strongly divergent between Carrion and Hooded Crows, and it carries all but one of the fixed allelic differences we detected. Modified after [Poelstra et al. 2014](/publication/poelstra-genomic-2014/index.html) and  [Vijay et al. 2016](/publication/vijay-evolution-2016/index.html)." >}}
